File: patterns/Python/pattern3.py
"""
Generate the following pattern
1
1 2
1 2 3
1 2 3 4
1 2 3 4 5
"""

def pattern3(n):
    for i in range(1,n+1):
        for j in range(i):
            print(j+1," ", end="")
        print("")
        
def main():
    i = int(input("Enter the number of rows: "))
    print("Here is the generated pattern\n")
    pattern3(i)
    
if __name__ == "__main__":
    main()

File: patterns/C++/pattern3.cpp
/*
Generate the following pattern
1
1 2
1 2 3
1 2 3 4
1 2 3 4 5
*/

#include<bits/stdc++.h>
using namespace std;

void pattern3(int n){
    for(int i=1; i<=n; i++){
        for(int j=1; j<i+1; j++){
            cout << j << " ";
        }
        cout << "\n";
    }
}

int main() {
    int n;
    cout << "Enter the number of rows(n): ";
    cin >> n;
    cout << "Here is the generated pattern\n";
    pattern3(n);

}